<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj zdjęcia</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPicMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>831</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPicPrev">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPicNext">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>410</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextPic">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>520</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Następne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrevPic">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Poprzednie</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>130</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxInfo">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokazuj informacje</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWatermark_app">
    <property name="title">
     <string>Watermark app</string>
    </property>
   </widget>
   <addaction name="menuWatermark_app"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
